tests/beacon/test_helpers.py

```diff
@@ -480,31 +483,25 @@ def mock_get_shard_committees_at_slot(state,
             )
 
 
-def test_get_active_validator_indices(sample_validator_record_params):
+def test_get_active_validator_indices(sample_validator_record_params, far_future_slot):
+    current_slot = 1
     # 3 validators are ACTIVE
     validators = [
         ValidatorRecord(
             **sample_validator_record_params,
         ).copy(
-            status=ValidatorStatusCode.ACTIVE,
+            activation_slot=0,
+            exit_slot=far_future_slot,
         )
         for i in range(3)
     ]
-    active_validator_indices = get_active_validator_indices(validators)
+    active_validator_indices = get_active_validator_indices(validators, current_slot)
     assert len(active_validator_indices) == 3
 
-    # Make one validator becomes ACTIVE_PENDING_EXIT.
     validators[0] = validators[0].copy(
-        status=ValidatorStatusCode.ACTIVE_PENDING_EXIT,
+        activation_slot=2,  # activation_slot < current_slot
```

Comment:
The comment here is incorrect -- "activation_slot > current_slot"

I think the following will be clearer
```suggestion
        activation_slot=current_slot + 1,  # activation_slot > current_slot
```